common/app-starters-micrometer-common/src/main/java/org/springframework/cloud/stream/app/micrometer/common/SpringCloudStreamMicrometerCommonTags.java

```diff
@@ -28,21 +29,21 @@
  * instance index and guids. Later are necessary to allow discrimination and aggregation of app metrics by external
  * metrics collection and visualizaiton tools.
  *
+ * Use the management.metrics.cloud.stream.app.common.tags.enabled=false property to disable inserting those tags.
+ *
  * @author Christian Tzolov
  */
 @Configuration
+@ConditionalOnProperty(name = "management.metrics.cloud.stream.app.common.tags.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
should check is we can use a property that is already used in boot starting with `management.metrics`....